collector/pg_stat_user_tables.go

```diff
@@ -180,7 +186,8 @@ var (
 		autovacuum_count,
 		analyze_count,
 		autoanalyze_count,
-		pg_total_relation_size(relid) as total_size
+		pg_total_relation_size(relid) as total_size,
+		pg_table_size(relid) as table_only_size
```

Comment:
Hmm, I wonder if we should refactor this.

> pg_total_relation_size accepts the OID or name of a table or toast table, and returns the total on-disk space used for that table, including all associated indexes. This function is equivalent to pg_table_size + pg_indexes_size.

Instead of exposing the `pg_total_relation_size`, we expose the two underlying metrics and let the users sum up the values themselves.

```suggestion
		pg_indexes_size(relid) as indexes_size,
		pg_table_size(relid) as table_size
```